.clang-format

```diff
@@ -20,7 +20,7 @@ BinPackArguments: true
 BinPackParameters: true
 BraceWrapping:
   AfterClass: true
-  AfterCaseLabel: true
+  #AfterCaseLabel: true
```

Comment:
Why'd you comment this out?

Reply:
I need to comment it out in order for my clang-format to work and I forgot to exclude changes.